### ISSUE TITLE
Use Pinboard search instead of a list of tags (was: It looks like the api can't work well)

### ISSUE THREAD

**Issue by linhuiw** [nhoizey/PinboardInGoogle#9]
I use this plugin in firefox, but it seems always get empty result in google result page;

I find the api in pinboard docs, i can't find this api, it seems this api can not work well; 
Do you have the docs about this Api?

// Generate Pinboard API JSON URL
// https://feeds.pinboard.in/json/v1/secret:....../u:nhoizey/t:24-70mm/t:canon?count=20

**Comment by nhoizey** [comment]
You have to put your own Pinboard secret API token, which you will find on https://pinboard.in/settings/password

If you go to this page, the userscript should get the token automaticaly.

I plan to migrate this userscript to a proper Web Extension soon.

**Comment by linhuiw** [comment]
![image](https://cloud.githubusercontent.com/assets/722603/15250020/3a83c4f4-1955-11e6-8fbb-cd2e32dfb9e4.png)
yes , i have add my secret and u, the result is empty. 
but in my pinboard serach page it has results.

![image](https://cloud.githubusercontent.com/assets/722603/15250056/654f51f8-1955-11e6-9da0-61c55f385ceb.png)

so i think this api does not work well.

**Comment by linhuiw** [comment]
i know the problem. 
in your code, you translate queryString to a list of tags, i did not add this tag in my pinboard. 

may be first i think the search can match the title, not the tag

**Comment by nhoizey** [comment]
You're right, it might be a better idea to use the search directly, instead of trying to build a collection of tags. Less accurate maybe, but with probably more results.

**Comment by whispy** [comment]
Is there any update on this? It would be nice if, in addition to just searching titles, we could also do full text search too (like can be done by adding '&fulltext=1' to the search URL).
